Index: vala-0.12.0/src/vapi/gstreamer-0.10.vapi
===================================================================
--- vala-0.12.0.orig/src/vapi/gstreamer-0.10.vapi	2010-12-20 14:07:20.000000000 -0600
+++ vala-0.12.0/src/vapi/gstreamer-0.10.vapi	2011-04-26 10:51:28.772216174 -0600
@@ -297,7 +297,7 @@
 		[CCode (cname = "gst_clock_new_single_shot_id")]
 		public ClockID.single_shot (Gst.Clock clock, Gst.ClockTime time);
 		public void unschedule ();
-		public Gst.ClockReturn wait (Gst.ClockTimeDiff jitter);
+		public Gst.ClockReturn wait (out Gst.ClockTimeDiff? jitter);
 		public Gst.ClockReturn wait_async (Gst.ClockCallback func);
 		public Gst.ClockReturn wait_async_full (Gst.ClockCallback func, GLib.DestroyNotify destroy_data);
 	}
@@ -681,9 +681,9 @@
 		public void parse_error (out GLib.Error gerror, out string? debug);
 		public void parse_info (out GLib.Error gerror, out string? debug);
 		public void parse_new_clock (out unowned Gst.Clock clock);
-		public void parse_qos (bool live, uint64 running_time, uint64 stream_time, uint64 timestamp, uint64 duration);
-		public void parse_qos_stats (Gst.Format format, uint64 processed, uint64 dropped);
-		public void parse_qos_values (int64 jitter, double proportion, int quality);
+		public void parse_qos (out bool live, out uint64 running_time, out uint64 stream_time, out uint64 timestamp, out uint64 duration);
+		public void parse_qos_stats (out Gst.Format format, out uint64 processed, out uint64 dropped);
+		public void parse_qos_values (out int64 jitter, out double proportion, out int quality);
 		public void parse_request_state (out Gst.State state);
 		public void parse_segment_done (out Gst.Format format, out int64 position);
 		public void parse_segment_start (out Gst.Format format, out int64 position);
